bmi_lstm.py

```diff
@@ -41,10 +41,10 @@ def __init__(self):
         'model_name':         'LSTM for Next Generation NWM',
         'version':            '1.0',
         'author_name':        'Jonathan Martin Frame',
-        'grid_type':          'scalar', # JG Edit
-        'time_step_size':      1,       # JG Edit
-        #'time_step_type':     'donno', # JG Edit (unused)  
-        #'step_method':        'none',  # JG Edit (unused)
+        'grid_type':          'scalar', 
+        'time_step_size':      1,       
+        #'time_step_type':     'donno', #unused  
+        #'step_method':        'none',  #unused
         #'time_units':         '1 hour' #NJF Have to drop the 1 for NGEN to recognize the unit
         'time_units':         'hour' }
```

Comment:
Slightly confusing with `self._time_units = "s"`

Reply:
From what I can tell, seems like the only item actually being used from `_att_map` is `'model_name': 'LSTM for Next Generation NWM'`?